packages/ses/test/error/test-tame-v8-error-unsafe.js

```diff
@@ -51,6 +51,28 @@ function simulateDepd() {
   return site.name;
 }
 
+test('SES compartment error compatibility', t => {
+  const c1 = new Compartment();
+  const result = c1.evaluate(`
+    function prepareObjectStackTrace(_, stack) {
+      return stack;
+    }
+    const limit = Error.stackTraceLimit;
+    const obj = {};
+    const prep = Error.prepareStackTrace;
+    Error.prepareStackTrace = prepareObjectStackTrace;
+    Error.stackTraceLimit = Math.max(10, limit);
+    // capture the stack
+    Error.captureStackTrace(obj);
+    // slice this function off the top
+    const stack = obj.stack.slice(1);
```

Comment:
This test case goes well beyond what we agreed was an adequate solution, IIUC. Just as the powerless SharedError currently has a do-nothing accessor for `stackTraceLimit` that just ignores the assignment silently, I thought we agreed that a similar do-nothing accessor for `prepareStackTrace` that just ignores the assignment silently, would be adequate. However, this test requires the existence of a callable `Error.captureStackTrace` that causes an effect dependent on an earlier assignment to `prepareStackTrace`. That would be a global communications channel. Such a SharedError constructor would not be powerless.

Please revise this example so that it would pass with the minimal acceptable solution to the problem. Either that, or help me correct my misunderstanding of the problem.